### PR TITLE
Support pay by link feature

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.4.5]
+
+**Improvement**
```
Comment (emicha) [inline CHANGELOG.md:6]: Shouldn't it say feature instead?

**Review by emicha — COMMENTED** [review]
Missing documentation explaining how to use the new feature

**Review by emicha — CHANGES_REQUESTED** [review]
Missing an integration test validating that the feature works.

File: CHANGELOG.md (hunk 1 above)
Comment (BushraAsif) [reply CHANGELOG.md]: I've followed the change according to this documentation
https://keepachangelog.com/en/1.0.0/#how

**Comment by BushraAsif** [comment]
> Missing an integration test validating that the feature works.

We have created a separate ticket for this.

**Comment by BushraAsif** [comment]
> Missing documentation explaining how to use the new feature

Update wiki.md file once approved I will update the wiki file

**Review by emicha — APPROVED** [review]

**Pull request merged**